xscen/utils.py

```diff
@@ -522,11 +533,29 @@ def get_cat_attrs(
 @parse_config
 def maybe_unstack(
     ds: xr.Dataset,
-    coords: str = None,
-    rechunk: bool = None,
+    coords: Optional[str] = None,
+    rechunk: Optional[bool] = None,
```

Comment:
Same here

Reply:
Good catch. After speaking with Juliette, this turns out to be a type error. It should be `Optional[dict]`, not `bool`.